### PR TITLE
feat: Added detailed information on amount of pubdata per write

### PR DESCRIPTION
**Pull request by mm-zk**
# What :computer: 
* Display detailed information on how many pubdata bytes does each write cost
* added "paid" option to --show-storage-logs to see only the writes that cost pubdata

# Why :hand:
* The cost of pubdata is in many cases the highest cost related to the transaction
* with boojum release, we use pubdata compression, so each slot might use between 6 and 64 bytes - so it is important for users to see how much they have to pay - and this would also be a good signal for them to see where they can save.

# Evidence :camera:
image

### CODE REVIEW

File: src/node/in_memory.rs [hunk 1]
```diff
@@ -177,9 +179,10 @@ impl FromStr for ShowStorageLogs {
             "none" => Ok(ShowStorageLogs::None),
             "read" => Ok(ShowStorageLogs::Read),
             "write" => Ok(ShowStorageLogs::Write),
+            "paid" => Ok(ShowStorageLogs::Paid),
```
Comment (MexicanAce) [inline src/node/in_memory.rs:182]: What is `Paid`?

File: src/node/in_memory.rs [hunk 2]
```diff
@@ -74,7 +75,7 @@ pub const TEST_NODE_NETWORK_ID: u32 = 260;
 /// L1 Gas Price.
 pub const L1_GAS_PRICE: u64 = 50_000_000_000;
 /// L2 Gas Price (0.25 gwei).
```
Comment (MexicanAce) [inline src/node/in_memory.rs:77]: Let's update this comment to match

Comment (mm-zk) [reply src/node/in_memory.rs:77]: my bad - this change to l2 gas price should go in separate PR

File: src/node/in_memory.rs (hunk 1 above)
Comment (mm-zk) [reply src/node/in_memory.rs:182]: storage logs slots for which you have to pay. Any ideas for a better names are welcome.

Comment (MexicanAce) [reply src/node/in_memory.rs:182]: This is AWESOME actually <3

**Review by MexicanAce — APPROVED** [review]
LGTM

**Pull request merged**